src/main/java/me/eccentric_nz/TARDIS/ARS/TARDISARSInventory.java

```diff
@@ -52,13 +53,15 @@ public TARDISARSInventory(TARDIS plugin) {
         // direction pad up
         ItemStack pad_up = new ItemStack(Material.CYAN_WOOL, 1);
         ItemMeta up = pad_up.getItemMeta();
+        assert up != null;
```

Comment:
Yes, technically `getItemMeta()` could return null, but if you can find me a case where this code would actually generate a NPE I'd be very surprised...

Reply:
Agree here, age-old Bukkit deprecation that may actually harm us if not solved at the source. `assert` is also generally frowned upon in favor of solving it as it is a band-aid fix that will force a problem to occur.

Reply:
I was unaware of that usage of `assert`, `Objects.requireNonNull`, and `Objects.equals` were frowned upon, so I can remove those. Or, rather, I can remake the fork and not add them.